views/pages/admin/image-store.pug

```diff
@@ -0,0 +1,29 @@
+extends /views/templates/admin-base.pug
+
+append styles
+    link(href="/assets/css/admin/image-store.css" rel="stylesheet")
+
+append scripts 
+    script(src="/assets/js/admin/image-store.js" defer)
+
+block body
+  div(id="content")
+    h1 MathSoc Images
+    div(class="actions")
+        input(type="text" name="image-search-input" placeholder="Search for images...")
+        form(method="POST" action="/api/image/upload" enctype="multipart/form-data")
+            input(type="file" name="image" required multiple)
```

Comment:
```suggestion
            input(type="file" name="image" required multiple accept="image/*")
```
Although the backend already prevents the user from uploading bad files, we should also prevent them from even seeing the option.